libs/novui/src/json-schema-components/widgets/InputWidget.tsx

```diff
@@ -0,0 +1,21 @@
+import { getInputProps, WidgetProps } from '@rjsf/utils';
+import { Input } from '../../components/input/Input';
+
+export const InputWidget = (props: WidgetProps) => {
+  const { type, value, label, schema, onChange, options, required, readonly, rawErrors, disabled } = props;
+  const inputProps = getInputProps(schema, type, options);
+
+  return (
+    <Input
+      description={props.schema.description}
+      onChange={(event) => onChange(event.target.value)}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Do we need to call `preventDefault` or `stopPropagation`? Or does that break functionality?‏